src/herder/HerderImpl.cpp

```diff
@@ -1033,7 +1034,16 @@ HerderImpl::triggerNextLedger(optional<uint32_t> ledgerSeqToTrigger,
     }
     if (*ledgerSeqToTrigger > slotIndex)
     {
+        if (!usingManualClose)
+        {
+            throw std::logic_error(
+                "Allowing sequence number jump without MANUAL_CLOSE");
+        }
         slotIndex = *ledgerSeqToTrigger;
+        LedgerTxn updateLastSeqnoToPredecessor(mApp.getLedgerTxnRoot());
```

Comment:
Nitpick: perhaps just `ltx` (that's the name we conventionally use everywhere) unless you have some compelling reason to have a long detailed name?

Reply:
The code that did this has moved; it's called `ltx` there now.